pkg/plugin/plugin.go

```
@@ -151,6 +151,15 @@ func splitIDs(s string) []string {
 	return ids
 }
 
+// cloneIDs returns a copy of the id list, so a caller can reorder or filter it
+// without disturbing the values already bound to the prepared statement.
+func cloneIDs(ids []string) []string {
```

golangci-lint (2.12.2)

[error] func cloneIDs is unused

(unused)